src/util/musicUtils.js

```diff
@@ -199,3 +217,48 @@ export const triads = [
   new Chord("Duuri", [[MAJOR, THIRD], [PERFECT, FIFTH]]),
   new Chord("Ylinouseva", [[MAJOR, THIRD], [AUGMENTED, FIFTH]]),
 ]
+
+export const createRandomInterval = () => {
+  const quality = qualities[randomInt(0, qualities.length - 1)]
```

Comment:
randomInt parameter max is exclusive, so this should be randomInt(0, qualities.length)